### PR TITLE
[8.0] fix for pylint 3.2.0 

### PR DESCRIPTION
**Pull request by chaen**
BEGINRELEASENOTES
*All
FIX: fix pylint 3.2.0 warnings

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/Core/Base/AgentModule.py [hunk 1]
```diff
@@ -329,6 +329,7 @@ def _setShifterProxy(self):
 
     def am_go(self):
         # Set the shifter proxy if required
+        initialWallTime = initialCPUTime = cpuPercentage = -1
```
Comment (chrisburr) [inline src/DIRAC/Core/Base/AgentModule.py]: I'm not sure I like adding this for the sake of a linter...

I think I'd prefer refactoring it like this:

```diff
         if self.activityMonitoring:
             initialWallTime, initialCPUTime, mem = self._startReportToMonitoring()
-        cycleResult = self.__executeModuleCycle()
-        if self.activityMonitoring and initialWallTime and initialCPUTime:
+            cycleResult = self.__executeModuleCycle()
             cpuPercentage = self._endReportToMonitoring(initialWallTime, initialCPUTime)
+.       else:
+            cycleResult = self.__executeModuleCycle()
```

File: src/DIRAC/ConfigurationSystem/scripts/dirac_admin_check_config_options.py [hunk 2]
```diff
@@ -174,6 +174,8 @@ def _printDiff(self, entry, level=""):
             diffType, entryName, _value, changes, _comment = entry
         elif len(entry) == 4:
             diffType, entryName, _value, changes = entry
+        else:
+            raise ValueError(f"Invalid entry {entry}")
```
Comment (chrisburr) [inline src/DIRAC/ConfigurationSystem/scripts/dirac_admin_check_config_options.py:178]:
```suggestion
            raise NotImplementedError(f"Invalid entry {entry}")
```

File: src/DIRAC/Core/Security/VOMS.py [hunk 3]
```diff
@@ -74,6 +75,7 @@ def getVOMSAttributes(self, proxy, switch="all"):
                     if value and value not in attributes and value in validVOMSAttrs:
                         attributes.append(value)
 
+        returnValue = None
```
Comment (chrisburr) [inline src/DIRAC/Core/Security/VOMS.py]: Why not add `raise NotImplementedError(switch)`?

File: src/DIRAC/Core/Base/AgentModule.py (hunk 1 above)
Comment (chaen) [reply src/DIRAC/Core/Base/AgentModule.py]: Works with me. I am surprise that pylint does not have a false positive on that one though :-)

File: src/DIRAC/ConfigurationSystem/scripts/dirac_admin_check_config_options.py (hunk 2 above)
Comment (chaen) [reply src/DIRAC/ConfigurationSystem/scripts/dirac_admin_check_config_options.py:178]: The definition of `ValueError` seems more fitting:
> Raised when an operation or function receives an argument that has the right type but an inappropriate value

File: src/DIRAC/Core/Security/VOMS.py (hunk 3 above)
Comment (chaen) [reply src/DIRAC/Core/Security/VOMS.py]: I am happy with that too. This argument is actually never used :shrug:

**Review by fstagni — APPROVED** [review]

**Pull request merged**